volatility3/framework/plugins/linux/pagecache.py

```diff
@@ -0,0 +1,504 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+import math
+import logging
+import datetime
+from dataclasses import dataclass, astuple
+from typing import List
+
+from volatility3.framework import renderers, interfaces
+from volatility3.framework.renderers import format_hints
+from volatility3.framework.interfaces import plugins
+from volatility3.framework.configuration import requirements
+from volatility3.plugins import timeliner
+from volatility3.plugins.linux import mountinfo
+
+vollog = logging.getLogger(__name__)
+
+
+@dataclass
```

Comment:
Do these belong here, or should they live in LinuxUtilities somewhere (ie, are they always likely to be used with this plugin, or should they be more general), given that there are already elf and bash extesions under `framework/symbols/linux/extensions` so support for adding new files and common types already exists (albeit, it does require tinkering with `__init__.py` to import them be default, I think).

Reply:
Hm I don't think so. These dataclasses are specific for this plugin and its output. The idea of these two is similar to the MVC design pattern. Being InodeInternal and InodeUser the Model and View representations respectively. 
They work great! The code becomes a lot simpler and clearer.